touchin-pagination/Code/View/view_controller.swift.liquid

```diff
@@ -0,0 +1,73 @@
+import UIKit
+import LeadKit
+import TableKit
+
+final class {{ module_info.name }}ViewController: UIViewController {
+
+    var viewModel: {{ module_info.name }}ViewModel!
+
+    @IBOutlet private weak var tableView: UITableView!
+
+    private lazy var tableDirector = TableDirector(tableView: tableView)
+
+    private lazy var paginationWrapper = {
```

Comment:
same about shortening

Reply:
Too long
<img width="831" alt="screen shot 2018-03-27 at 17 52 09" src="https://user-images.githubusercontent.com/6436245/37975312-af73a64c-31e7-11e8-89f5-143006de34f0.png">
or ugly
<img width="557" alt="screen shot 2018-03-27 at 17 52 31" src="https://user-images.githubusercontent.com/6436245/37975313-af8e68ce-31e7-11e8-9bd8-72e03fe38064.png">

what did you prefer? :)

Reply:
:)